CMakeLists.txt

```diff
@@ -110,7 +110,7 @@ INCLUDE_DIRECTORIES(${HDF5_INCLUDE_DIR})
 ADD_SUBDIRECTORY(carta-protobuf)
 ADD_SUBDIRECTORY(carta-scripting-grpc)
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
-INCLUDE_DIRECTORIES(/opt/carta-casacore/include /opt/carta-casacore/include/casacore /opt/carta-casacore/include/casacode)
+INCLUDE_DIRECTORIES(SYSTEM /opt/carta-casacore/include /opt/carta-casacore/include/casacore /opt/carta-casacore/include/casacode)
```

Comment:
what does the `SYSTEM` label do?

Reply:
Sorry, I didn't mean to push that change. Here, `SYSTEM` turns off the warnings from external libraries, and we get a lot of warning clutter from casacore.

Reply:
I just dropped this change. I like adding SYSTEM while working on something particular, but I think it's better to always keep all those warnings visible.